### PR TITLE
Onboard Argus credentials: delete and list commands

### PR DESCRIPTION
**Pull request by DiogoFerrao**

### CODE REVIEW

File: internal/cmd/argus/credentials/create/create.go [hunk 1]
```diff
@@ -18,15 +18,13 @@ import (
...
-	hidePasswordFlag = "hide-password"
+	instanceIdFlag = "instance-id"
 )
 
 type inputModel struct {
 	*globalflags.GlobalFlagModel
 
-	HidePassword bool
-	InstanceId   string
+	InstanceId string
```
Comment (vicentepinto98) [inline internal/cmd/argus/credentials/create/create.go:27]: Why remove the `hide-password` functionality?

File: internal/cmd/argus/credentials/delete/delete.go [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+				return err
+			}
+
+			instanceLabel, err := argusUtils.GetInstanceName(ctx, apiClient, model.InstanceId, model.ProjectId)
+			if err != nil {
+				instanceLabel = model.InstanceId
+			}
+
+			if !model.AssumeYes {
+				prompt := fmt.Sprintf("Are you sure you want to delete credentials %s of instance %q? (This cannot be undone)", model.Username, instanceLabel)
```
Comment (vicentepinto98) [inline internal/cmd/argus/credentials/delete/delete.go:62]:
```suggestion
				prompt := fmt.Sprintf("Are you sure you want to delete credentials for username %q of instance %q? (This cannot be undone)", model.Username, instanceLabel)
```

File: internal/cmd/argus/credentials/delete/delete.go [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			_, err = req.Execute()
+			if err != nil {
+				return fmt.Errorf("delete Argus credentials: %w", err)
+			}
+
+			p.Info("Deleted credentials %s of instance %q\n", model.Username, instanceLabel)
```
Comment (vicentepinto98) [inline internal/cmd/argus/credentials/delete/delete.go:76]:
```suggestion
			p.Info("Deleted credentials for username %q of instance %q\n", model.Username, instanceLabel)
```

File: internal/cmd/argus/credentials/create/create.go (hunk 1 above)
Comment (DiogoFerrao) [reply internal/cmd/argus/credentials/create/create.go:27]: In that sense, I also added documentation to the help of the command stating this

**Review by vicentepinto98 — APPROVED** [review]

**Pull request merged**